buildSrc/src/main/groovy/airbyte-connector-acceptance-test.gradle

```diff
@@ -19,7 +19,7 @@ class AirbyteConnectorAcceptanceTestPlugin implements Plugin<Project> {
                         '-w', "$targetMountDirectory",
                         '-e', "AIRBYTE_SAT_CONNECTOR_DIR=${project.projectDir.absolutePath}",
                         'airbyte/connector-acceptance-test:dev',
-                        '-p', 'integration_tests.acceptance',
+                        '--acceptance-test-config', targetMountDirectory,
```

Comment:
are you sure this isn't a red herring? I don't follow why this fixes the issue

Reply:
For example with https://github.com/airbytehq/airbyte/pull/22393 the acceptance tests run correctly when called this way but fail with “could not find module integration_tests.acceptance” with the old call.

I’m not sure why though, that’s why I pinged you on this.

Reply:
Ah I think I see the problem. 

`-p integration_tets.acceptance` tells the running pytest process (CAT in this case) to look at a file `integration_tests/acceptance.py` to configure tests. 

In this situation the working directory is always set to the connector's directory. Python connectors always have a `integration_tests/acceptance.py` file, so this works for them. Even the java connectors which implement CAT also have that directory with an `acceptance.py` file. 

That directory is important because it allows the user to configure setup/teardown actions e.g: create a database and tear it down. 

I think the correct way to handle this would be to create an `integration_tests/acceptance.py` directory for the connector in question, or maybe conditionally add this flag depending on whether that directory exists. 

I think the `--acceptance-test-config` arg is unnecessary. By default SAT looks for an `acceptance-test-config.yaml` in the working directory (which is always set to the connector dir) so I think we can remove this arg.